app/components/articles/document_component.html.erb

```diff
@@ -9,8 +9,13 @@
   class: classes.flatten.join(' ') do %>
   <%= header %>
   <div class="document-main-section">
-    <%= title %>
-    <%= document_metadata %>
+    <% if @document.eds_restricted? %>
```

Comment:
Shouldn't there be a test about who the user is too?

Reply:
Ideally using https://github.com/sul-dlss/SearchWorks/pull/4744 😉

Reply:
I would assume no? The eds_restricted tests the title and if the title is the eds restricted pattern (eds title is restricted for guests) https://github.com/sul-dlss/SearchWorks/blob/54f901e3734ff2b43ead7648b2844b5cc6af3ce5/app/models/eds_document.rb#L85. I would assume that if the user is logged in, eds would not return that title?

Reply:
Thanks. I missed that.